cmd/version.go

```diff
@@ -1,22 +1,16 @@
 package cmd
 
 import (
-	"context"
 	"github.com/liatrio/gh-trusted-builds-attestations/build"
+	"github.com/spf13/cobra"
 )
 
-type Version struct {
-}
-
-func (v *Version) Init(ctx context.Context, flags []string) error {
-	return nil
-}
-
-func (v *Version) Is(s string) bool {
-	return "version" == s
-}
-
-func (v *Version) Run() error {
-	println(build.Version)
-	return nil
+func VersionCmd() *cobra.Command {
```

Comment:
Do we still need this command? https://github.com/spf13/cobra/blob/main/user_guide.md#version-flag

Reply:
I kept it for parity and made the output of `attestation --version` match this command. I think it's a common convention with Go programs to have `version` as a subcommand